Unity-Technologies/Graphics
# [XPipeline][URP]Camera Editor - Move all GUIContents to a dedicated class

### Purpose of this PR
Unify the code of camera editor in both pipelines,

---
### Testing status
- Check that the camera type values are correct.
- Check that the fields, secitions appear in the same way

---
### Comments to reviewers
- This intermediate step towards HDRP and URP camera editor unification.
- Moving Styles private class to `UniversalRenderPipelineCameraUI.skin.cs`
- Changelog not updated because everything is internal.
